### PR TITLE
Amendment to Rule 203

### PR DESCRIPTION
**Pull request by ArchmageInc**
This is designed to help speed changes along and insure the previously agreed blue line is adhered to.

### CODE REVIEW

**Comment by ArchmageInc** [comment]
Just as it says. Only the votes present are counted, though rule 105 states:

`**105.** Every player is an eligible voter. Every eligible voter must participate in every vote on rule-changes.`

**Comment by ArchmageInc** [comment]
I plan to change rule 105 though.

**Comment by jimmyhmiller** [comment]
So, since that rule takes precedence, I would guess that means right now proposals don't close after 24 hours but instead once everyone has voted.

**Comment by ArchmageInc** [comment]
That is how I read it.

File: laws.md [hunk 1]
```diff
@@ -84,5 +82,10 @@ New Judges are not bound by the decisions of old Judges. New Judges may, however
 
 This rule takes precedence over every other rule determining the winner.
 
-**301.** Ammendment of rule **201.**: Each player may submit one proposal every twenty-four (24) hours, in any player order. Each proposal will remain for the 24 hour period to be voted on and discussed. At the end of the 24 hour period, the rule is either adopted, or rejected.
+**301.** Each player may submit one proposal every twenty-four (24) hours, in any player order. Each proposal will remain for the 24 hour period to be voted on and discussed. At the end of the 24 hour period, the rule is either adopted, or rejected.
```
Comment (jimmyhmiller) [inline laws.md]: Maybe this is nitpicky. But this is modifying to rules now. The rule was put in with that text and shouldn't be changed unless you want to propose a rule change.

File: laws.md [hunk 2]
```diff
@@ -84,5 +82,10 @@ New Judges are not bound by the decisions of old Judges. New Judges may, however
...
 
+**303.** At the end of a proposal's 24 hour period, if the number of yay's is greater than the nay's the proposal is considered adopted.
```
Comment (jimmyhmiller) [inline laws.md]: You might want to add a clause in here that a vote is either a yay or nay. Then you get all the benefits of other laws that deal with votes. Right now how it is written people could leave multiple yay's or nay's.

File: laws.md (hunk 1 above)
Comment (ArchmageInc) [inline laws.md]: Yes, it is nitpicky, but it isn't modifying the other rule, it is a formatting change as per your request for rules to not reference other rules.

Comment (ArchmageInc) [inline laws.md]: removed

File: laws.md (hunk 2 above)
Comment (ArchmageInc) [inline laws.md]: Right now, 207 prevents that

**Comment by jimmyhmiller** [comment]
Alright, I could nitpick more, but I actually do have some substantive things I'm not sure about with the rule.

I don't like the idea of the proposer not voting. Maybe I propose a rule and someone convinces me it is a bad idea. Shouldn't I then be able to vote against it?

**Comment by ArchmageInc** [comment]
You would be able to modify the rule in such a way that you would like it, no?

**Comment by jimmyhmiller** [comment]
What if they comment after the vote has been opened? No way to change it then. At least that's what I see as implied by this sentence in #111.

> The proponent decides the final form in which the proposal is to be voted on and, unless the Judge has been asked to do so, also decides the time to end debate and vote.

If you decide that it is to be voted on, it is the final form.

**Comment by ArchmageInc** [comment]
Oh for fuck's sake. Then the idea that votes are void if a change is made would make no sense!

**Pull request closed without merging**